cli/commands.js

```diff
@@ -108,7 +108,7 @@ const defaultCommands = {
             },
         ],
         desc: `Create entities from the JDL file/URL/content passed in argument.
-  By default everything is run in series. If you like to run things faster use '--fork' flag to run in parallel.`,
+  By default everything is run in parallel for new applications and in series when there is any existing application in the folder. If you like to run things faster use '--fork' flag to run in parallel or '--interactive' flag to run in series.`,
```

Comment:
```suggestion
  When re-generating multiple applications in same jdl consider using fork option.`,
```
Or drop it, IMO is just complicates everything, the regeneration is not too long to worth.